TOPIC6-REST-API/src/main/java/classes/Meeting.java

```diff
@@ -0,0 +1,73 @@
+package classes;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+@Entity
+public class Meeting {
+
+	@OneToMany(mappedBy = "meeting")
+	private List<Attendee> attendees;
+
+	@ManyToOne
+	private Room room;
+
+	@Id
+	@GeneratedValue
+	private Long id;
+
+	Meeting() { // jpa only

```

Comment:
A good practice (talking about readability) is to define all the class attributes first and then the constructors